sdk/eventhub/event-hubs/package.json

```diff
@@ -29,7 +36,8 @@
   "files": [
     "dist/",
     "dist-esm/src/",
-    "typings/event-hubs.d.ts",
+    "types/latest/",
+    "types/3.1/",
```

Comment:
Just came across this..
![image](https://user-images.githubusercontent.com/10452642/80845755-0eb55280-8bbf-11ea-9241-0dbb560e8e70.png)

We might have missed it in the other packages..
https://github.com/Azure/azure-sdk-for-js/blob/008095baaee075d78dcce2ce214f89afc2ebf5af/sdk/core/core-amqp/package.json#L27

https://github.com/Azure/azure-sdk-for-js/blob/008095baaee075d78dcce2ce214f89afc2ebf5af/sdk/servicebus/service-bus/package.json#L41

Reply:
Hmm, we should log an issue to add those, and maybe also see what impact that has on our docs. I'm betting that TypeDoc either doesn't need it or doesn't use it. I see some other packages like keyvault also only include the bundled types and not the metadata.

Reply:
Oh interesting, this seems to be used to by api-extractor so that when it's parsing a package and needs to look at a dependency's types, it can know what dialect of tsdoc was used in the dependency:
https://github.com/microsoft/tsdoc/issues/7

Reply:
Created https://github.com/Azure/azure-sdk-for-js/issues/8662 to track.

Reply:
Nice, thanks for logging.